src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -88,7 +91,12 @@ export const ArtworkFilter: React.FC<
       onChange={onChange}
       ZeroState={ZeroState}
     >
-      <ArtworkFilterRefetchContainer viewer={viewer} {...rest} />
+      <ArtworkFilterRefetchContainer
+        viewer={viewer}
+        artistId={artistId}
+        artistName={artistName}
```

Comment:
**suggestion**: instead of passing `artistName`, `artistId`, we can pass `viewer`. In the future, we will be able to extract all the values we need from the `viewer` directly and this will make it easier for us to implement the "saved search" button on other artwork grids (artist series, collection, etc).